app/backend/express-server.js

```diff
@@ -0,0 +1,91 @@
+const crypto = require('crypto');
+const express = require('express');
+const logger = require('morgan');
+const compress = require('compression');
+const session = require('express-session');
+const helmet = require('helmet');
+const lusca = require('lusca');
+const cookieParser = require('cookie-parser');
+const bodyParser = require('body-parser');
+const connectPgPool = require('./setup-pg');
+const pgQuery = require('./queries');
+const { countApplication } = require('./queries/application');
```

Comment:
I don't see this defined...

Reply:
sorry, there's 4 lines here. Which are you referring to?